lib/src/view/opening_explorer/opening_explorer_screen.dart

```diff
@@ -0,0 +1,845 @@
+  });
+
+  final String pgn;
+  final AnalysisOptions options;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final ctrlProvider = analysisControllerProvider(pgn, options);
+    final canGoBack =
+        ref.watch(ctrlProvider.select((value) => value.canGoBack));
+    final canGoNext =
+        ref.watch(ctrlProvider.select((value) => value.canGoNext));
+
+    return Container(
+      color: Theme.of(context).platform == TargetPlatform.iOS
+          ? CupertinoTheme.of(context).barBackgroundColor
+          : Theme.of(context).bottomAppBarTheme.color,
+      child: SafeArea(
+        top: false,
+        child: SizedBox(
+          height: kBottomBarHeight,
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceAround,
+            children: [
+              Expanded(
+                child: BottomBarButton(
+                  label: context.l10n.menu,
+                  onTap: () {
+                    _showAnalysisMenu(context, ref);
+                  },
+                  icon: Icons.menu,
+                ),
+              ),
+              Expanded(
+                child: BottomBarButton(
+                  label: context.l10n.analysis,
+                  onTap: () => pushReplacementPlatformRoute(
+                    context,
+                    builder: (_) => AnalysisScreen(
+                      pgnOrId: pgn,
+                      options: options,
+                    ),
+                  ),
+                  icon: Icons.biotech,
+                ),
+              ),
+              Expanded(
+                child: RepeatButton(
+                  onLongPress: canGoBack ? () => _moveBackward(ref) : null,
+                  child: BottomBarButton(
+                    key: const ValueKey('goto-previous'),
+                    onTap: canGoBack ? () => _moveBackward(ref) : null,
+                    label: 'Previous',
+                    icon: CupertinoIcons.chevron_back,
+                    showTooltip: false,
+                  ),
+                ),
+              ),
+              Expanded(
+                child: RepeatButton(
+                  onLongPress: canGoNext ? () => _moveForward(ref) : null,
+                  child: BottomBarButton(
+                    key: const ValueKey('goto-next'),
+                    icon: CupertinoIcons.chevron_forward,
+                    label: context.l10n.next,
+                    onTap: canGoNext ? () => _moveForward(ref) : null,
+                    showTooltip: false,
+                  ),
+                ),
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  void _moveForward(WidgetRef ref) =>
+      ref.read(analysisControllerProvider(pgn, options).notifier).userNext();
+  void _moveBackward(WidgetRef ref) => ref
+      .read(analysisControllerProvider(pgn, options).notifier)
+      .userPrevious();
+
+  Future<void> _showAnalysisMenu(BuildContext context, WidgetRef ref) {
```

Comment:
Should be renamed `_showExplorerMenu` perhaps? And also we could avoid code duplication with the analysis menu here I think.